internal/sqldb/sqlquery/get_import_table.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlquery
+
+import (
+	"database/sql"
+	"encoding/json"
+	"time"
+
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/internal/util"
+)
+
+// Represents "metadata" column of import table
+type ImportMetadata struct {
+	NumObs  int32
```

Comment:
Wondering if we want to allow them to be `nil` by making them pointers? We might add more metadata in the future and only a subset of fields may have values at different times.

Reply:
Good idea, made them pointers.